superset/db_engine_specs/base.py

```diff
@@ -1433,6 +1434,11 @@ def get_parameters_from_uri(
         cls, uri: str, encrypted_extra: Optional[Dict[str, Any]] = None
     ) -> BasicParametersType:
         url = make_url(uri)
+        query = {
+            key: value
+            for (key, value) in url.query.items()
+            if (key, value) not in cls.encryption_parameters.items()
```

Comment:
Do you need to test that both key and value match? Is there a case where key will match but not the value?